cmake/find_rdkafka.cmake

```diff
@@ -1,4 +1,6 @@
-option (ENABLE_RDKAFKA "Enable kafka" ON)
+if (NOT ARCH_ARM)
+    option (ENABLE_RDKAFKA "Enable kafka" ON)
```

Comment:
Is it Ok that Kafka is disabled on ARM?
Kafka engine and ARM is used in CloudFlare.